### PR TITLE
pull-test2 with more useful delta'd content

### PR DESCRIPTION
**Pull request by cgwalters**

### CODE REVIEW

**Comment by jlebon** [comment]
bot, retest this please

File: src/libostree/ostree-repo-static-delta-compilation.c [hunk 1]
```diff
@@ -912,13 +912,11 @@ process_one_bsdiff (OstreeRepo                       *repo,
        * try to reverse engineer things more in the "show" path, but that gets
        * hard/messy as it's quite optimized for execution now.
        */
-#if 0
```
Comment (jlebon) [inline src/libostree/ostree-repo-static-delta-compilation.c]: Were those removed by accident, I suppose?

File: tests/libtest.sh [hunk 2]
```diff
@@ -267,6 +269,50 @@ setup_fake_remote_repo1() {
...
+    mkdir -p usr/{etc,bin,lib,share}
+    ln -sr usr/bin bin
+    ln -sr usr/lib lib
+    tar xf ${test_srcdir}/fah-deltadata-old.tar.xz
+    remote_ref=exampleos/42/x86_64/main
+    cd ..
+    ${CMD_PREFIX} ostree --repo=${test_tmpdir}/ostree-srv/repo commit \
+                  --consume $commit_opts --add-metadata-string version=42.0 -b ${remote_ref} \
+                  --tree=dir=files
+    rm files -rf
```
Comment (jlebon) [inline tests/libtest.sh]: Nit: we shouldn't have to do this with `--consume`, right?

**Comment by cgwalters** [comment]
Rebased 🏄 and fixup ⬆️ (nice catches thanks!)

**Comment by jlebon** [comment]
@rh-atomic-bot r+ 3e235b6

**Pull request closed without merging**